blocks/media-upload-button/index.js

```diff
@@ -121,25 +142,6 @@ class MediaUploadButton extends Component {
 		onSelect( multiple ? attachment : attachment[ 0 ] );
 	}
 
-	onOpen() {
```

Comment:
I guess this means we should also set a selection object when we're not in a gallery (Editing an image block is broken with this change).

Reply:
The problem was fixed with the new commit I added :)